### ISSUE TITLE
v6.11.0 breaks Custom Axios instance

### ISSUE THREAD

**Issue by NimmLor** [anymaniax/orval#725]
## What are the steps to reproduce this issue?

1. Set a custom axiosInstance via 
```ts
override: {
  mutator: {
    name: 'axiosMutator',
    path: './axiosMutator.ts',
  }
},
target: './generated/orval.ts',
```
2. Run orval
3. Look at the output

Example 2:
```ts
override: {
  mutator: {
    name: 'axiosMutator',
    path: './utils/axiosMutator.ts',
  }
},
target: './generated/orval.ts',
```

## What happens?

The above example generates:
```ts
import { axiosMutator } from '..axiosMutator'
import type { ErrorType } from '..axiosMutator'
```

Example 2 output uses `\` instead of `/`
```ts
import { axiosMutator } from '..\utils\axiosMutator';
import type { ErrorType } from '..\utils\axiosMutator';
```

## What were you expecting to happen?
The above example should:
```ts
import { axiosMutator } from '../axiosMutator'
import type { ErrorType } from '../axiosMutator'
```

Example 2 expected output: 
```ts
import { axiosMutator } from '../utils/axiosMutator';
import type { ErrorType } from '../utils/axiosMutator';
```

## Any logs, error output, etc?

Folder structure:

![image](https://user-images.githubusercontent.com/32486857/211283417-9bd8dae0-0c40-4911-bcf1-ce2948d1266c.png)

Git diff after the upgrade from `v6.11.0-alpha.4` to `v6.11.0`

![image](https://user-images.githubusercontent.com/32486857/211281237-4abd5c96-4c74-4abc-a7fb-df782084d4aa.png)

## Any other comments?

I've been using `v6.11.0-alpha.4` and many versions before that quite some time now without any issues.

## What versions are you using?

**Operating System:** Windows 11
**Package Version:** 6.11.0
**Browser Version:** not relevant

**Comment by anymaniax** [comment]
Hello @NimmLor, it's a problem on windows I am currently adding a fix

**Comment by anymaniax** [comment]
Hello @NimmLor, can you try if `6.11.1-alpha.1` is working for you

**Comment by NimmLor** [comment]
That didn't work 😅 

The path mentioned in the error is correct.

```
🍻 Start orval v6.11.1-alpha.1 - A swagger client generator for typescript
node:internal/process/promises:279
            triggerUncaughtException(err, true /* fromPromise */);
            ^

[UnhandledPromiseRejection: This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason "failed to load from C:/development/monorepo/packages/api/orval.config.ts => TypeError: n is not a function".] {
  code: 'ERR_UNHANDLED_REJECTION'
}
```

**Comment by anymaniax** [comment]
Sorry for that just published another version that fixes that problem too => `6.11.1-alpha.2`. If you can confirm that it works I will do a proper release with it

**Comment by NimmLor** [comment]
Works as expected! Thanks!

**Comment by anymaniax** [comment]
Fixed with `6.11.1`

**Comment by melloware** [comment]
Just an FYI I bumped to Axios 1.2.3 today from 1.2.2 and I had to change my Mutator from AxiosRequestConfig to RawAxiosRequestConfig as they change the TypeScript def to make headers required.

```js
export const useAxiosMutator = <T>(): ((config: RawAxiosRequestConfig) => Promise<T>) => {
	return (config: RawAxiosRequestConfig) => {
```
